### DISCUSSION TITLE
Ability to group services in servicemap by additional tags

### DISCUSSION THREAD

**Discussion by rlex** [grafana/grafana#52183] [Observability]
We have multiple lokis configured on same cluster. stg/prod/etc. Problem is, in loki i can't override JAEGER_SERVICE_NAME. So all lokis will be reporting under same name, which, i assume, will be problematic for service maps. I can theoretically pass some additional tag (ie namespace, since each loki is deployed to separate namespace) but i can't seem to be able to group those service maps using additional tags.

It's possible to adjust dimensions in tempo to add additonal labels, but it will be much easier to just use tags from traces since those can be easily added at deploy time, ie namespace tag.

Related slack discussion https://grafana.slack.com/archives/C01D981PEE5/p1657597542016969

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
